RELEASE.md

```diff
@@ -117,6 +120,25 @@ Promotion should occur in two steps:
 
 **NOTE**: The promotion of wheels to PyPI can only be done once so take caution when attempting to promote wheels to PyPI, (see https://github.com/pypa/warehouse/issues/726 for a discussion on potential draft releases within PyPI)
 
+# Doing a Patch Release
+
+A patch release is a maintenance release of PyTorch that includes fixes for regressions found in a previous minor release.
+
+## Things to consider for a patch release
+
+1. Does the regression break core functionality?
```

Comment:
Would it be possible to explicitly extend "core functionality" to also include the domain libraries, which aren't part of "core"?

Reply:
We could use the same process for sure for domain libs. 

Btw, I would probably think about something like overall 'patch' releases are specific to addressing bugs and regressions between major releases. This could include:
1. An unexpected change that broke user facing functionality; and/or
2. A significant performance regression was discovered;

Patch releases are NOT for adding new functionality, API surface or progressing feature maturity.

Reply:
cc @gchanan anything to add here?

Reply:
Just to clarify, my comment here is about making "breaking a domains library" a good enough reason for having a bugfix pytorch release. Currently the text only states "core functionality" and it's not obvious whether the domains are part of those "core functionalities" or not.

I fully agree that bugfix releases should not include new features.

Reply:
yes agree. If we break a domain lib, I would consider this something urgent to fix and should be included.

Reply:
```suggestion
1. Does the regression break core functionality?
2. Does the regression break first party domain libraries that depend on locked versions of `pytorch`?
    * [pytorch/vision](https://github.com/pytorch/vision)
    * [pytorch/audio](https://github.com/pytorch/audio)
    * [pytorch/text](https://github.com/pytorch/text)
```